File: 1.4/1-4.js
const sorter = (str) => str.split('').sort().join('')

const reverser = (str) => str.split('').reverse().join('')

const pPC = (str1, str2) => {
    console.log
    if (str1 === reverser(str1) && str2 === reverser(str2)) {
        if (sorter(str1) === sorter(str2)) {
            return true
        } 
    }
    return false
}

console.log(pPC('aba', 'aba'))

File: 1.4/1-4.py
def pal_perm_checker(str1, str2):
    if str1 == str1[::-1] and str2 == str2[::-1]:
        if ''.join(sorted(str1)) == ''.join(sorted(str2)):
            return True
    return False


print(pal_perm_checker('thingniht', 'thniginht'))